addon-test-support/@ember/test-helpers/dom/-is-focusable.js

```diff
@@ -0,0 +1,15 @@
+import isFormControl from './-is-form-control';
+import isContentEditable from './-is-content-editable';
+
+const FOCUSABLE_TAGS = ['LINK', 'A'];
```

Comment:
I really don't know why I listed `LINK` as a focusable tag originally. I think it was a mindfart, it should be gone.